src/components/VideoOverview/VideoOverview.tsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react';
+import { getDateDiff, getVideoDate, VideoDate } from '../../helpers/date';
+import { clipTitle } from '../../helpers/titles';
+import VideoEmbed from '../VideoEmbed';
+import { StyledContainer, StyledContent, StyledImg, StyledMeta, StyledPlayIcon, StyledTitle } from './styles';
+import { Video } from './types';
+
+export default function VideoOverview({ item }: { item: Video }) {
+  const [publishedAgo, setPublishedAgo] = useState<VideoDate>();
+  const [loadVideo, setLoadVideo] = useState<boolean>(false);
```

Comment:
is there any reason it's named 'loaded'? maybe show?

Reply:
Because it loads from the url passed down to iframe? :P If you think show is more suitable I will change it